### DISCUSSION TITLE
5.2.0

### DISCUSSION THREAD

**Discussion by JJJ** [stuttter/ludicrousdb#172] [General]
## What's Changed
* Update composer/installers dependency requirement by @SteelWagstaff in https://github.com/stuttter/ludicrousdb/pull/136
* Correct the `dbh` property type by @johnbillion in https://github.com/stuttter/ludicrousdb/pull/132
* Minor cleanup and fix for key with value 0 by @kkmuffme in https://github.com/stuttter/ludicrousdb/pull/125
* Update class-ludicrousdb.php by @seed-glauco in https://github.com/stuttter/ludicrousdb/pull/141
* Remove apostrophe from it's by @bobbingwide in https://github.com/stuttter/ludicrousdb/pull/142
* Issue/147 - improve compat with PHP 8.1 and newer by @JJJ in https://github.com/stuttter/ludicrousdb/pull/152
* Update composer development dependencies by @vladislavbailovic in https://github.com/stuttter/ludicrousdb/pull/146
* Issue/139: Add Try/Catch to: _do_query() by @JJJ in https://github.com/stuttter/ludicrousdb/pull/154
* Remove use_mysqli checks. by @JJJ in https://github.com/stuttter/ludicrousdb/pull/153
* Ignore development files in export archives by @ocean90 in https://github.com/stuttter/ludicrousdb/pull/155
* Replace "master/slave" with "primary/replica" by @JJJ in https://github.com/stuttter/ludicrousdb/pull/169
* Improve persistent object cache compatibility (wp redis) by @JJJ in https://github.com/stuttter/ludicrousdb/pull/119
* Reimplement check_connection() by @JJJ in https://github.com/stuttter/ludicrousdb/pull/171

## New Contributors
* @SteelWagstaff made their first contribution in https://github.com/stuttter/ludicrousdb/pull/136
* @johnbillion made their first contribution in https://github.com/stuttter/ludicrousdb/pull/132
* @seed-glauco made their first contribution in https://github.com/stuttter/ludicrousdb/pull/141
* @bobbingwide made their first contribution in https://github.com/stuttter/ludicrousdb/pull/142
* @vladislavbailovic made their first contribution in https://github.com/stuttter/ludicrousdb/pull/146
* @ocean90 made their first contribution in https://github.com/stuttter/ludicrousdb/pull/155

**Full Changelog**: https://github.com/stuttter/ludicrousdb/compare/4.2.0...5.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/stuttter/ludicrousdb/releases/tag/5.2.0'>5.2.0</a>.</em>

**Comment by archon810** [comment]
Somehow after updating to this release, all reads started going to the master, which quickly killed it. No reads were going to the slaves. Any idea why this update could have caused it?

**Reply by JJJ** [reply]
Uh oh. Not off the top of my head; no. I'll try to replicate.

Can you share your relevant config in a new discussion thread in the [Help Category](https://github.com/stuttter/ludicrousdb/discussions/categories/help)?

**Reply by archon810** [reply]
@JJJ I posted some details here https://github.com/stuttter/ludicrousdb/discussions/182.
